probot-app-report-benchmarks/index.js

```diff
@@ -124,8 +130,21 @@ module.exports = app => {
           labels: [`head`, `base`],
           data: [head_json.data[i].time, base_json.data[i].time],
         };
+        titles.push(head_json.data[i].build);
+        let headMedian = median(head_json.data[i].time);
+        let baseMedian = median(base_json.data[i].time);
+        let deltaMedian = headMedian - baseMedian;
+        if (Math.abs(deltaMedian) > baseMedian * 0.1) {
```

Comment:
Minor: can we move `0.1` into a named variable or add a comment?  Fairly important value that could use the additional context.